test/unit/spec/EnrollChoices_spec.js

```diff
@@ -16,27 +17,79 @@ define([
   'helpers/xhr/MFA_ENROLL_multipleOktaVerify',
   'helpers/xhr/SUCCESS'
 ],
-function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, Router,
+function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, FactorUtil, Router,
   $sandbox, resAllFactors, resAllFactorsOnPrem, resPush, resMultipleU2F,
   resMultipleOktaVerify, resSuccess) {
 
   var { $, _ } = Okta;
   var itp = Expect.itp;
   var tick = Expect.tick;
   var factorEnrollList = {
-    'QUESTION': 0,
-    'GOOGLE_AUTH': 1,
-    'OKTA_VERIFY': 2,
-    'RSA_SECURID': 3,
-    'SYMANTEC_VIP': 4,
-    'YUBIKEY': 5,
-    'SMS': 6,
-    'CALL': 7,
-    'DUO': 8,
-    'WINDOWS_HELLO': 9,
-    'U2F': 10,
-    'GENERIC_SAML': 11,
-    'GENERIC_OIDC': 12
+    'QUESTION': {
+      index:      0,
+      provider:   'OKTA',
+      factorType: 'question'
+    },
+    'GOOGLE_AUTH': {
+      index:      1,
+      provider:   'GOOGLE',
+      factorType: null
+    },
+    'OKTA_VERIFY': {
+      index:      2,
+      provider:   'OKTA',
+      factorType: 'token:software:totp'
+    },
+    'RSA_SECURID': {
+      index:      3,
+      provider:   'RSA',
```

Comment:
For multiple enrollments, we do not use provider. So please remove provider. See `getFactorNameForFactorType` in FactorUtil